src/tools/schema2kotlin.ts

```diff
@@ -237,7 +240,16 @@ ${imports.join('\n')}
 
     for (const connection of particle.connections) {
       const handleName = connection.name;
-      const entityType = entityTypeName(particle.name, connection.name);
+      const capitalHandleName = connection.name[0].toUpperCase() + connection.name.slice(1);
+      const ng = nodeGenerators.find(generator => {
+        const kg = <KotlinGenerator>generator.generator;
+        return kg.node.connections.includes(capitalHandleName);
+      });
```

Comment:
Generally acronyms aren't a great choice for variable names (this isn't Go!), except _maybe_ for very short lambdas. Could you rename these? `nodeGenerator` and `kotlinGenerator`?

Reply:
Fixed.